frontend/public/components/namespace.jsx

```diff
@@ -247,14 +247,13 @@ class NamespaceDropdown_ extends React.Component {
     const onChange = newNamespace => dispatch(UIActions.setActiveNamespace(newNamespace));
 
     return <div className="co-namespace-selector">
-      <span>{model.label}:</span>
       <Dropdown
         className="co-namespace-selector__dropdown"
         menuClassName="co-namespace-selector__menu"
         noButton
         canFavorite
         items={items}
-        title={title}
+        title={`${model.label}:   ${title}`}
```

Comment:
I think the extra spaces will be collapsed unless we add a `white-space` style or maybe HTML whitespace entities.

We could add a titlePrefix prop if we want to style it differently, but in that case we'd want to handle the button case. Your original fix only added it for noButton.